S7.Net/Types/S7String.cs

```diff
@@ -9,6 +9,11 @@ namespace S7.Net.Types
     /// </summary>
     public static class S7String
     {
+        /// <summary>
+        /// The Encoding used when serializing and deserializing S7String (Encoding.ASCII by default)
+        /// </summary>
+        public static Encoding StringEncoding { get; set; }
```

Comment:
```suggestion
        public static Encoding StringEncoding { get; set; } = Encoding.ASCII;
```
* avoids the `null`-check in L38, and L64 (whole line can be deleted)
* doesn't need a static constructor (they aren't ideal, because they don't use `beforefieldinit` (in IL))

Reply:
Yes, I agree. Initially, the checks in L38 and L64 were intended to avoid null exception if the user later sets the StringEncoding to null. But then, maybe it's ok to throw an exception in that case

Reply:
Ah, good point. Maybe this should be a "regular property" where the setter has the null-check and if `null` throw an ArgumentNullException?
This would be the "most correct"* implementation.

\* except concerns about race-conditions, but assuming this static property will be set once at startup this shouldn't be a problem at all.

Reply:
Excellent! I will change that now